nba_py/__init__.py

```diff
@@ -1,16 +1,30 @@
 from requests import get
-from datetime import datetime
+from datetime import datetime, timedelta
 from nba_py.constants import League
+import os
 
 HAS_PANDAS = True
 try:
     from pandas import DataFrame
 except ImportError:
     HAS_PANDAS = False
 
+HAS_REQUESTS_CACHE = True
```

Comment:
What you mean "fast-forward your repo to include those commits instead of re-adding the code?" ?